Add rosé pine dawn and moon themes

JaninaWibker opened the pull request:
This adds the dawn and moon themes (both source files and minified files) and updates the readme accordingly.

I updated `rosepine.css` to use css variables. The 2 new files are essentially just copies of it with different variables.

I'm not sure whether or not including screenshots for the readme.md file in the repo is what you want, if you'd like to I can update it.

JaninaWibker committed:
add rosepine-dawn and rosepine-moon themes

This adds the rosé pine dawn and moon themes by updating the css of
rosepine.css to use css variables instead of directly referencing the
hex colors.
For the 2 new themes separate files are added, basically being
duplicates of rosepine.css with updated css variables.

JaninaWibker committed:
update readme

Updates the `## Usage` section to include references to the newly added
`rosepine-dawn.min.css` and `rosepine-moon.min.css`

JaninaWibker committed:
update Gallery in readme

This updates the `## Gallery` section in the readme file.
It adds screenshots of how the rosepine, rosepine-moon and rosepine-dawn
themes would look like in a codeblock.

As I don't know how I would generate the same image as was previously in
the readme file for the other themes I decided to replace all with new
screenshots.

mvllow commented:
Thank you for the new variants! If you wouldn't mind, please do remove the screenshots from the repo and use the GitHub cdn. Drag an image into any textarea on GitHub, e.g. in these comments, for a link.

Other than that, I would like to rename rosepine in the filenames to rose-pine but I am happy to do that in a follow-up PR because what you did matches what we have currently :)

JaninaWibker commented:
I can update the filenames and remove the assets folder, no problem!

One concern I have is that renaming `rosepine.min.css` would break other peoples code as the (previous) link in the readme file wouldn't work anymore.
Should I rename all files including the files in the `min/` folder or just the files in the base directory?

JaninaWibker committed:
replace assets folder with github cdn links

JaninaWibker committed:
rename: rosepine -> rose-pine

This renames the files in the base directory from `rosepine*.css` to
`rose-pine*.css`

mvllow commented:
Yea, I was afraid of that – honestly I wouldn't recommend anyone to use that link for a production site as it's too fragile. This should either be removed (in my opinion) or a warning placed above to not use that link in production. Again, I don't mind making any of these changes either but I do appreciate all the work you're putting into this :)

I know this isn't the best indicator of who is using this but [searching "rosepine.min.css"](https://github.com/search?q=rosepine.min.css&type=code) throughout GitHub only comes with relative links, none pointing to this repo.

JaninaWibker committed:
rename: `min/rosepine*.css` -> `min/rose-pine*.css`

This renames the files in the `min/` directory as well.
A note about possible future updates to the links listed in the readme
has also been added.

JaninaWibker commented:
I renamed the files in the `min/` directory as well and added a small note to the readme file about possible changes to the links

mvllow commented on readme.md:
```diff
@@ -16,17 +16,48 @@
 ## Gallery
 
-### As seen on https://stepbro.voring.me
+### Rosé Pine
```
```suggestion
**Rosé Pine**
```

mvllow commented on readme.md:
```diff
@@ -16,17 +16,48 @@
-### As seen on https://stepbro.voring.me
+### Rosé Pine
+
+![rose-pine](https://user-images.githubusercontent.com/16372321/227316298-6668954d-859e-439d-8a1d-2fb79346c100.jpg)
```
```suggestion
![highlight.js with Rosé Pine](https://user-images.githubusercontent.com/16372321/227316298-6668954d-859e-439d-8a1d-2fb79346c100.jpg)
```

mvllow commented on readme.md:
```diff
@@ -16,17 +16,48 @@
+![rose-pine](https://user-images.githubusercontent.com/16372321/227316298-6668954d-859e-439d-8a1d-2fb79346c100.jpg)
+
+
+### Rosé Pine Moon
```
```suggestion
**Rosé Pine Moon**
```

mvllow commented on readme.md:
```diff
@@ -16,17 +16,48 @@
+
+### Rosé Pine Moon
+
+![rose-pine-moon](https://user-images.githubusercontent.com/16372321/227316293-579c772a-ab7b-4b8f-ae19-4bafc61b0c99.jpg)
```
```suggestion
![highlight.js with Rosé Pine Moon](https://user-images.githubusercontent.com/16372321/227316293-579c772a-ab7b-4b8f-ae19-4bafc61b0c99.jpg)
```

mvllow commented on readme.md:
```diff
@@ -16,17 +16,48 @@
+![rose-pine-moon](https://user-images.githubusercontent.com/16372321/227316293-579c772a-ab7b-4b8f-ae19-4bafc61b0c99.jpg)
+
+
+### Rosé Pine Dawn
```
```suggestion
**Rosé Pine Dawn**
```

mvllow commented on readme.md:
```diff
@@ -16,17 +16,48 @@
+### Rosé Pine Dawn
 
-![image](https://user-images.githubusercontent.com/44733677/156076709-f01a5624-d50a-4e09-9bae-48494802ef07.png)
+![rose-pine-dawn](https://user-images.githubusercontent.com/16372321/227316267-5d98c0ed-2046-47f1-98a6-213588297abd.jpg)
```
```suggestion
![highlight.js with Rosé Pine Dawn](https://user-images.githubusercontent.com/16372321/227316267-5d98c0ed-2046-47f1-98a6-213588297abd.jpg)
```

mvllow reviewed (changes requested):
A few nitpicks and this should be good to go :)

JaninaWibker committed:
Apply suggestions from code review

Co-authored-by: not <mvllow@icloud.com>

mvllow commented:
Thanks again for all your time/work put into this! Lovely PR 💜
